pyOCD/tools/flash_tool.py

```diff
@@ -144,12 +154,42 @@ def main():
             if args.hide_progress:
                 progress = None
 
+            # Convert address and count to int.
+            if args.address:
+                args.address = int(args.address, base=0)
```

Comment:
Instead of converting address to an integer here you could add type checking to the args above.  That way argparse could print its own error message.  Something like "type=lambda val:int(val, 0)" in add_argument.  Or if that is too long you could define the conversion function explicitly.  It would probably also be good to use this same function for args, count and skip